### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by Aduda-Boaz**
This PR is for initializing projects with components. The milestone is complete and works as required.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,18 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Nav />
+      <Routes>
+        <Route path="bookstore" element={<BookDisplay />} />
```
Comment (TedLivist) [inline src/App.js]: - [x] The **BookDisplay** component here should be your root route. Kindly change this route path here to `'/'`.

  ```
  <Route path="/" element={<BookDisplay />} />
  ```

File: src/components/NavBar.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    <header className="Navbar">
+      <nav className="Nav-header">
+        <h1 className="Nav-logo">
+          <NavLink to="/bookstore/" rel="noopener noreferrer">
+            Bookstore CMS
+          </NavLink>
+        </h1>
+        <ul>
+          <li>
+            <NavLink to="/bookstore/" rel="noopener noreferrer">
```
Comment (TedLivist) [inline src/components/NavBar.js]: - [x] Kindly change this **`to`** attribute here to match the root path defined in the App component.

File: src/components/BookList.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        </div>
+      </div>
+      <div className="Progress">
+        <p>Current Chapter</p>
+        <p>Chapter 17</p>
+        <button type="button" className="progressBtn btn">Update Book</button>
+      </div>
+    </div>
+  );
+}
```
Comment (TedLivist) [inline src/components/BookList.js]: - [x] There should be a component representing a **single** book. The component should take in book data as a prop and render the **`title`** and **`author`**. If this component is intended for that then kindly refactor accordingly.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Aduda-Boaz,

Good job so far!
✔️ Good documentation
✔️ React-router-dom works well.
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/NavBar.js (hunk 2 above)
Comment (Aduda-Boaz) [reply src/components/NavBar.js]: Change made

File: src/App.js (hunk 1 above)
Comment (Aduda-Boaz) [reply src/App.js]: change made

File: src/components/BookList.js (hunk 3 above)
Comment (Aduda-Boaz) [reply src/components/BookList.js]: Change made

**Review by t-yanick — APPROVED** [review]
Hi @Aduda-Boaz, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**